backend/src/main/scala/bloop/io/AbsolutePath.scala

```diff
@@ -30,4 +32,8 @@ object AbsolutePath {
   def apply(path: String)(implicit cwd: AbsolutePath): AbsolutePath = apply(NioPaths.get(path))(cwd)
   def apply(path: Path)(implicit cwd: AbsolutePath): AbsolutePath =
     if (path.isAbsolute) new AbsolutePath(path) else cwd.resolve(path.toString)
+
+  // Necessary to test wrong paths in tests...
+  private[bloop] def completelyUnsafe(path: String): AbsolutePath =
```

Comment:
Wouldn't `unsafe` or `toUnsafe` sound a bit less scary?

Reply:
It's meant to be super scary, so that anyone but me uses it :stuck_out_tongue: This makes an absolute path out of something it's not an absolute path.

Reply:
Haha got it, didn't think it was _that_ scary